HydroModelBuilder/ModelInterface/flopyInterface/viz/MT3D_PP_viz.py

```diff
@@ -176,8 +177,8 @@ def viewConcsByZone(self, nper='all', specimen=None):
     multiplier = 1.
     fig = plt.figure(figsize=(width * multiplier, height * multiplier))
 
-    vmin = np.amin(conc[conc > 0.])  # 0.0
-    vmax = np.amax(conc)  # 100.0
+    vmin = np.amin(conc[conc > 0.])
+    vmax = np.amax(conc)
```

Comment:
vmax gets set as the axis maximum for the array, but then is never used. Instead its value gets assigned to `max_conc` and the variable name is reused to represent `100.0` in Line 210

Reply:
@ConnectedSystems , yes I had some bad values and did some trickery to be able to see the distribution of concs better. Again, needs a new home ... or I'll see what is actually generic enough to keep and get rid of the rest